README.adoc

```diff
@@ -268,26 +275,26 @@ and `system/src/main/liberty/config/server.xml` files:
 <feature>usr:opentracingZipkin-0.30</feature>
 ----
 
-Make sure your services are running, then simply point your browser to any of their endpoints and
+Make sure your services are running. Then simply point your browser to any of their endpoints and
```

Comment:
We want a comma after "Then." "Then, simply point..."

Reply:
Ok, fixed